scripts/compose.py

```diff
@@ -1815,6 +1820,14 @@ def init_start_parser(self, parser, services, argv=None):
             default=DEFAULT_APM_SERVER_URL,
         )
 
+        parser.add_argument(
```

Comment:
`--apm-server-secret-token` is already an option, should use that instead.

Reply:
note you'll need to merge master in following #292

Reply:
thanks. I've reverted my changes. There's still the separate option for `--secret-token` for sourcemap uploads as it is. I decided not to try and change that in this PR.